core/arch/arm/include/sm/optee_smc.h

```diff
@@ -290,6 +290,12 @@
 #define OPTEE_SMC_SEC_CAP_HAVE_RESERVED_SHM	(1 << 0)
 /* Secure world can communicate via previously unregistered shared memory */
 #define OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM	(1 << 1)
+/*
```

Comment:
Actually the UNREGISTERED_SHM relates the the so-called default shm (`NSEC_SHM` in the core). This shm is "negotiated" at core inits with nsec world (some SMCs allow to get/set the nsec_shm location, but core_mmu.c expects it from static config `CFG_SHMEM_START/_SIZE`). It happens to be physically contiguous and as a single memory chunk. That's its real attributes.